pandas/io/tests/parser/usecols.py

```diff
@@ -254,3 +254,25 @@ def test_usecols_with_parse_dates_and_usecol_names(self):
                            usecols=[3, 0, 2],
                            parse_dates=parse_dates)
         tm.assert_frame_equal(df, expected)
+
+    def test_usecols_accepts_unicode_strings(self):
+        # see gh-13219
+
+        s = u'''AAA,BBB,CCC,DDD
+        0.056674973,8,True,a
+        2.613230982,2,False,b
+        3.568935038,7,False,a
+        '''
+
+        data = {
+            'AAA': {
+                0: 0.056674972999999997,
+                1: 2.6132309819999997,
+                2: 3.5689350380000002
+            },
+            'BBB': {0: 8, 1: 2, 2: 7}
+        }
+        expected = DataFrame(data)
+
+        df = self.read_csv(StringIO(s), usecols=[u'AAA', u'BBB'])
```

Comment:
What's the approach then?  Recode unicode values to UTF8 if the dtype is mixed in the [_validate_usecols_arg](https://github.com/pydata/pandas/blob/c9ffd7891dadd6e5590695e142f77a3476b5c4e3/pandas/io/parsers.py#L843)? or just accept 'mixed' type if they are all string_type?